scipy/optimize/nonlin.py

```diff
@@ -1350,7 +1350,7 @@ class KrylovJacobian(Jacobian):
         preconditioners. For example,
 
         >>> jac = BroydenFirst()
-        >>> kjac = KrylovJacobian(inner_M=jac.inverse).
+        >>> kjac = KrylovJacobian(inner_M=jac.inverse)
```

Comment:
I'm not sure what is this. `BroydenFirst().inverse` does not exist. Should it be `jac.aspreconditioner`?